addons/website_slides/tests/test_attendee.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.website_slides.tests import common
+
+
+class TestAttendee(common.SlidesCase):
+
+    def test_course_atttendee(self):
```

Comment:
``test_course_attendee_copy`` maybe ? And remember to add a small docstring to explain the bugfix you are testing, so that people know what this test is about without having to retro-engineer the test code :)